assets/queries/openAPI/2.0/security_requirement_not_defined_in_security_definition/test/negative2.yaml

```diff
@@ -0,0 +1,35 @@
+swagger: "2.0"
+info:
+  title: Simple API Overview
+  version: 1.0.0
+paths:
+  "/":
+    get:
+      operationId: listVersionsv2
+      summary: List API versions
+      responses:
+        "200":
+          description: 200 response
+      parameters:
+        - name: limit2
+          in: body
+          description: max records to return
+          required: true
+          schema:
+            type: object
+security:
+  petstore_auth:
```

Comment:
Here should be `- petstore_auth:` (the field `security` is an array). The same happens in the other YAML samples